third_party/terraform/tests/resource_google_folder_iam_policy_test.go

```diff
@@ -12,37 +11,6 @@ import (
 	resourceManagerV2Beta1 "google.golang.org/api/cloudresourcemanager/v2beta1"
 )
 
-func TestAccFolderIamPolicy_basic(t *testing.T) {
```

Comment:
The `_update` test was a superset, so now it's the only test.

Reply:
Could we rename `_update` to `_basic`, then?

Reply:
Done!